core/lib/dal/src/blocks_dal.rs

```diff
@@ -2481,8 +2556,10 @@ impl BlocksDal<'_, '_> {
         Ok(())
     }
 
-    pub async fn insert_mock_l1_batch(&mut self, header: &L1BatchHeader) -> DalResult<()> {
-        self.insert_l1_batch(
+    pub async fn insert_mock_l1_batch(&mut self, header: &L1BatchHeader) -> anyhow::Result<()> {
+        self.insert_l1_batch(header.number, header.timestamp, BatchFeeInput::default())
```

Comment:
let's avoid using `BatchFeeInput::default()`, same for other occurrences in this file